packages/fxa-settings/src/components/BrandMessaging/index.tsx

```diff
@@ -125,7 +128,7 @@ const BrandMessaging = ({
             </div>
           </div>
         )}
-        <div className="absolute w-1/12 right-0 top-0 pt-3 start-0">
+        <div className="absolute w-1/12 rtl:left-0 ltr:right-0 top-0 pt-3">
```

Comment:
`rtl:left-0 ltr:right-0` is equivalent to `end-0`, if we want to simplify this :)

Reply:
The X icon is now flexbox, so no longer applicable. Good to know though.